### DISCUSSION TITLE
How to put a range of dates for published label?

### DISCUSSION THREAD

**Discussion by ayousuf23** [mmistakes/minimal-mistakes#2706] [Q&A] [answered]
I want to add a range of dates for the published label for items in the portfolio. For example, next to "Published" I want it to say "September-October 2020" instead of "October 1, 2020." How can I do this?

**Comment by mmistakes** [comment, marked as answer]
The published field is determined by `page.date` which you either specify directly in a post's YAML Front Matter, or Jekyll assigns it for you automatically based on the filename. e.g. `2020-10-07-my-post.md` would have a `page.date` of 2020-10-07 which [this line](https://github.com/mmistakes/minimal-mistakes/blob/master/_includes/page__date.html) runs through Jekyll's date filter here to generate `October 7, 2020`.

You could customize the theme, add your own YAML Front Matter like `published_range: "September-October 2020"` and then use something like this where you want the range to appear on the page:

```
Published: {{ page.published_range }}
```
